### DISCUSSION TITLE
Client sent an HTTP request to an HTTPS server.

### DISCUSSION THREAD

**Discussion by LukeVader-IV** [nextcloud/all-in-one#3607] [Questions] [answered]
When starting Nextcloud docker AIO for the first time, and filling in your valid domain for verification, It gives an unclear error in the UI saying to check container logs.

The container logs say the following:
`The response of the connection attempt to "http://{domain}:443" was: Client sent an HTTP request to an HTTPS server`

If ignored, I get stuck and can't reach the Nextcloud UI, after all the sub containers have been started. Instead get redirected back to the AIO interface.

The ports opened on the router are port 443, which is 8443 internally. And port 80, which is 8080 internally.

**Comment by LukeVader-IV** [comment, marked as answer]
the Nextcloud master container doesn't run the other necessary containers within it's own container, instead it starts up containers next to itself (which i didn't even know was possible).
This means that there are more ports being listened to then are specified in your compose file. The nextcloud container running next to the "mastercontainer" will always listen to port 80 (not 8080) and 443 (not 8443), ports over which you have no control over. ports 8080 and 8443 are _only_ for the AIO interface.
The solution is to make sure you forward port 443 from internal network (not 8443) to 443 on the external network.

**Reply by zzZzz-io** [reply]
I have the same problem but I don't know what you mean by forwarding the same port internally to externally.

**Reply by LukeVader-IV** [reply]
Simply make sure you are forwarding port 443 and port 80 on your router.
